.eslintrc.js

```diff
@@ -1,9 +1,5 @@
 const { config } = require('@dhis2/cli-style')
 
 module.exports = {
-    parser: 'babel-eslint',
-    extends: ['eslint:recommended', config.eslint],
-    rules: {
-        'max-params': ['error', { max: 5 }],
-    },
```

Comment:
We've got to somehow keep this part, because prop validation functions have 5 arguments.

Reply:
I tried this config in`.eslintrc.js`, which I believe is correct:
```javascript
module.exports = {
    extends: [config.eslint],
    rules: {	
        'max-params': ['error', { max: 5 }],	
    },
}
```

But somehow, running `yarn lint:style --verbose` with this setup is resulting in the following output:
```
yarn run v1.19.2
$ d2-style js check --all --verbose
d2-style > javascript
[DEBUG] Linting files: .commitlintrc.js, .eslintrc.js, .huskyrc.js, .prettierrc.js, babel.config.js, src/arrayWithLength.js, src/index.js, src/instanceOfComponent.js, src/mutuallyExclusive.js, src/requiredIf.js
.eslintrc.js
[WARNING] Code style issues found in the above file(s).
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```